python/src/uagents/network.py

```diff
@@ -221,7 +221,7 @@ def get_expiry(self, address: str) -> int:
 
         return (expiry - height) * AVERAGE_BLOCK_INTERVAL
 
-    def get_endpoints(self, address: str) -> List[AgentEndpoint]:
+    def get_endpoints(self, address: str) -> Optional[List[AgentEndpoint]]:
```

Comment:
Looks like this always returns a list.
```suggestion
    def get_endpoints(self, address: str) -> List[AgentEndpoint]:
```